src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -1988,10 +1988,11 @@
                             (some #(re-matches % field) (map re-pattern dotted-fields)))
                 {:node node
                  :state (conj state
-                              (format "'%s' is not a queryable object for %s, %s" field alias
-                                      (if (empty? qfields)
-                                        (format "%s has no queryable objects" alias)
-                                        (format "known queryable objects are %s" (json/generate-string qfields)))))}))
+                              (tru (format "''{0}'' is not a queryable object for {1}. %s"
+                                           (if (empty? qfields)
+                                             (format "Entity %s has no queryable objects" alias)
+                                             (format "Known queryable objects are %s"
+                                                     (utils/comma-separated-keywords qfields)))) field alias))}))
```

Comment:
I suspect this may not work translation-wise, i.e. tru won't be able to see all the text.